packages/web/src/scss/components/Collapse/index.html

```diff
@@ -104,7 +104,7 @@ <h2 class="docs-Heading">Mobile</h2>
       href="javascript:void(0)"
       data-toggle="collapse"
       data-target="CollapseExample1"
-      class="Button Button--secondary Button--medium d-mobile-none"
+      class="Button Button--secondary Button--medium d-none"
```

Comment:
This means the button is never shown 🙂. Shall we delete it then?

Plus, could you please provide other sections with an offset? Buttons are often too close to the adjacent text…

![obrazek](https://user-images.githubusercontent.com/5614085/196980096-623ab787-0b23-4136-a211-47c10a94e6f9.png)


![obrazek](https://user-images.githubusercontent.com/5614085/196979947-96f127af-e071-4909-ade9-2479b8f5ffa1.png)

Reply:
This is only for an example, because the assignment was this `data-breakpoint=mobile | tablet | desktop - breakpoint`, so in the final, the mobile is always hidden. But I can imagine a situation where this value could be triggered.

Offset was not an issue here, this is only for example, but you can implement it with whatever offset you want. This is only 'wrapper' for collapsible feature.

Only i can do i set break space under the button.

Reply:
OK, let's talk about it next week.

> Offset was not an issue here, this is only for example,

I know, but examples can look good too 🙂.